cardano-explorer/cardano-explorer.cabal

```diff
@@ -21,14 +21,19 @@ library
   default-language:     Haskell2010
   hs-source-dirs:       src
   ghc-options:          -Wall
+                        -Werror
                         -Wcompat
                         -fwarn-redundant-constraints
                         -fwarn-incomplete-patterns
                         -fwarn-unused-imports
                         -Wincomplete-record-updates
                         -Wincomplete-uni-patterns
                         -Weverything
-                        -- -Wno-unused-matches
+                        -Wno-safe
+                        -Wno-unsafe
+                        -Wno-implicit-prelude
+                        -Wno-error=partial-fields
+                        -Wno-error=unused-matches
```

Comment:
Better to manually add an leading unscore to specific values than enabling that everywhere.